bottlecap/src/events/mod.rs

```diff
@@ -15,9 +15,21 @@ impl MetricEvent {
     }
 }
 
+#[derive(Debug, Copy, Clone)]
+pub struct OomEvent {
+    pub timestamp: i64,
+}
+
+impl OomEvent {
+    #[must_use]
+    pub fn new(timestamp: i64) -> OomEvent {
+        OomEvent { timestamp }
+    }
+}
+
```

Comment:
Why not just make the OutOfMemory event be a `OutOfMemory(i64)`?

Reply:
I naively assumed we'd hang other metadata off of it, but we can make it an enum instead of a struct.

Reply:
Done